.artifact/tree.json

```diff
@@ -1,778 +1,364 @@
 {
-  "rimage": {
```

Comment:
I'm cutting artifacts which are not used by automated testing to speed up automated tests.

Reply:
@jeremyrhyde do you know if we need any of the files I'm propose we remove from the tree.json here? I see you are the most recent person who edited this file.